api/objects.proto

```diff
@@ -15,6 +15,8 @@ import "gogoproto/gogo.proto";
 message Meta {
 	// Version tracks the current version of the object.
 	Version version = 1 [(gogoproto.nullable) = false];
+	// SpecVersion tracks the current version of the spec.
+	Version spec_version = 4 [(gogoproto.nullable) = false];
```

Comment:
I don't think we should because the `Update*` RPCs have a single `Version` field right now, and I think it makes sense for them to have two `Version` fields so either spec versioning or object versioning or both can be validated.